runtimes/clone/src/lib.rs

```diff
@@ -248,6 +259,8 @@ impl pallet_aura::Config for Runtime {
 	type DisabledValidators = ();
 	type MaxAuthorities = MaxAuthorities;
 	type AllowMultipleBlocksPerSlot = ConstBool<false>;
+	#[cfg(feature = "experimental")]
+	type SlotDuration = ConstU64<SLOT_DURATION>;
```

Comment:
check if this is needed

Reply:
removed in: [50cb97a](https://github.com/KILTprotocol/kilt-node/pull/627/commits/50cb97a853c12795ff4a9d770d22aadf3f007a21)